components/VcItemContent.tsx

```diff
@@ -96,38 +110,35 @@ export const VcItemContent: React.FC<VcItemContentProps> = props => {
     <ImageBackground
       source={!props.verifiableCredential ? null : Theme.CloseCard}
       resizeMode="stretch"
-      borderRadius={4}
       style={
         !props.verifiableCredential
           ? Theme.Styles.vertloadingContainer
           : Theme.Styles.backgroundImageContainer
       }>
       <Column>
         <Row align="space-between">
-          <Row>
+          <Row margin="5 0 0 5">
             <ImageBackground
+              imageStyle={Theme.Styles.faceImage}
               source={
                 !props.verifiableCredential
                   ? Theme.ProfileIcon
                   : {uri: props.context.credential.biometrics.face}
               }
               style={Theme.Styles.closeCardImage}>
               {props.iconName && (
-                <Icon
-                  {...testIDProps('pinIcon')}
-                  name={props.iconName}
-                  type={props.iconType}
-                  color={Theme.Colors.Icon}
-                  style={{marginLeft: -80}}
-                />
+                <Image
+                  source={Theme.PinIcon}
+                  style={Theme.Styles.pinIcon}></Image>
               )}
             </ImageBackground>
-            <Column margin="6 0 0 10">
+
+            <Column margin="0 0 10 20" height={96} align="space-between">
               <Column>
                 <Text
                   testID="fullNameTitle"
-                  weight="bold"
-                  size="smaller"
+                  weight="regular"
+                  style={Theme.Styles.titleTextSize}
```

Comment:
Follow as mentioned above. Try to edit and reuse existing styles.

Reply:
Done